### PR TITLE
Installations: Add chrome shortcuts in additional resources section

### LINKED ISSUE #28236: Installations: Add a link to Chrome keyboard shortcuts in additional resources
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### Describe your suggestion

As during the course of the curriculum the students here often use browsers for plenty of tasks, Learning keyboard shortcuts for particularly the "Tabs & Windows" section in [Here](https://support.google.com/chrome/answer/157179?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Ctab-window-shortcuts) seems like it might be a good idea.

Though it could work in one of the later lessons too but I'm not sure where else it would work by not feeling _too_ out of place. 

Especially since this lesson is for _Installations_ but it's also the only lesson where Chrome is explictly mentioned.
image

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/paths/foundations/courses/foundations

### (Optional) Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by TheSeagullDev**
## Because

- Adds a link to Google Chrome shortcuts in installations lesson

## This PR

- Adds one additional resource with a link to common Google Chrome shortcuts for Mac and Linux

## Issue

Closes #28236

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/installations/installations.md [hunk 1]
```diff
@@ -502,4 +502,4 @@ The following questions are an opportunity to reflect on key topics in this less
 
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+- Google Chrome is one of the most important tools that you will be using throughout the curriculum. This [reference](https://support.google.com/chrome/answer/157179?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Ctab-window-shortcuts) has many of the common shortcuts that you may find useful using Chrome.
```
Comment (MaoShizhong) [inline foundations/installations/installations.md:505]: For accessibility, our [links should be sufficiently descriptive on their own](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#use-informative-titles). The below would preserve your wording but have the link text stand by itself.

```suggestion
- Google Chrome is one of the most important tools that you will be using throughout the curriculum. This reference has many [common shortcuts that you may find useful when using Chrome](https://support.google.com/chrome/answer/157179?hl=en&co=GENIE.Platform%3DDesktop#zippy=%2Ctab-window-shortcuts).
```

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
**OPTIONAL:** If you're happy to do so, could you also follow our [local linting and fixing instructions](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md#curriculum-linting) and run the `fix:lesson` script twice on the installations file to fix up some of the "surrounding blank lines" linting errors? There will still be an error at the end about "required heading structure" that you can ignore.

**Comment by TheSeagullDev** [comment]
@MaoShizhong I addressed the link text and also passed the lesson through the linter, let me know if anything further is needed.

**Review by MaoShizhong — APPROVED** [review]
:rocket: Thanks for contributing.

**Pull request merged**